src/utils/intlUtils.js

```diff
@@ -34,4 +34,4 @@ export const useTranslatedOptions = options => {
  */
 
 export const displayDateFromTimestamp = timestamp =>
-    moment.unix(timestamp).format('LTS');
+    timestamp ? moment.unix(timestamp).format('LTS') : '--';
```

Comment:
```suggestion
    timestamp ? moment.unix(timestamp).format('LTS') : textPlaceholder;
```

The place holder still needs to be imported